Applications/Scale/test/testScale.cpp

```diff
@@ -177,7 +185,13 @@ void scaleGait2354_GUI(bool useMarkerPlacement)
 
     // Compare ScaleSet
     ScaleSet stdScaleSet = ScaleSet(setupFilePath+"std_subject01_scaleSet_applied.xml");
-
+    // We used to compute a scale factor for the ground body, which does not make physical sense.
+    // Things connected to ground can be scaled, including their locations, but not the frame itself.
+    // Now that ground is its own frame it does not go through the body scale mill.
+    // We must remove it from the stdScaleSet.
```

Comment:
> Maybe all frames, including ground, should have scale methods.

This sounds good. I was playing with a toy example yesterday in which I had a muscle attached to ground. In 3.2, the attachment point would have scaled.